recipes/gimmemotifs/meta.yaml

```diff
@@ -10,11 +10,12 @@ source:
 
 build:
   skip: True  # [not py36]
-  number: 2
+  number: 3
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('gcc') }}
```

Comment:
You can remove this line. `{{ compiler('gcc') }}` is not valid and `{{ compiler('c') }}` already covers everythinh

Reply:
thanks